Kbs.Business.Tests/BoatType/BoatTypeEntityTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Kbs.Business.BoatType;
+
+public class BoatTypeEntityTests
+{
+
+    [Theory]
+    public void Properties_ShouldSetValues(int id, string name, BoatTypeRequiredExperience experience, int seats, int speed, bool hasSteeringWheel)
+    {
```

Comment:
xUnit1003: Theory methods must have test data. Use InlineData, MemberData, or ClassData to provide test data for the Theory.